sepal_ui/mapping/map_btn.py

```diff
@@ -22,10 +25,14 @@ def __init__(self, content, **kwargs):
         else:
             content = content[: min(3, len(content))].upper()
 
+        # create the style from default
+        style = json.loads((ss.JSON_DIR / "map_btn.json").read_text())
+        style.update(background=color.bg)
+
         # some parameters are overloaded to match the map requirements
         kwargs["color"] = "text-color"
         kwargs["outlined"] = True
-        kwargs["style_"] = " ".join([f"{k}: {v};" for k, v in map_btn_style.items()])
+        kwargs["style_"] = " ".join([f"{k}: {v};" for k, v in style.items()])
```

Comment:
Just a comment here. Wouldn't it be better if we set the colors of the button with the vuetify properties instead of changing the element CSS properties?